tests/test_virtualensemble.py

```diff
@@ -15,7 +15,7 @@
 
 try:
     # pylint: disable=unused-import
-    import pyarrow
+    import pyarrow  # noqa
```

Comment:
There is already a `pylint` statement here? Are both `pylint` and `flake8` used? Add `pylint` as well to CI?

Reply:
I think the pylint exception got in there for codacy's sake. 

LGTM is already in there, just not in the CI yaml file, and should cover pylint.

Pylint score is  currently "only" 9.86, and is hard to improve. Can't enforce pylint perfect score in CI before it is perfect.